utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/sinkvalue/SinkValueData.scala

```diff
@@ -33,21 +33,21 @@ object SinkValueData {
 
   case class SinkRecordValue(fields: ListMap[String, SinkValue]) extends SinkValue
 
-  // ParameterName can be created by concatenating names of records/object fields (RecordFieldName) - see AvroSinkValueParameter/JsonSinkValueParameter
+  // ParameterName can be created by concatenating names of records/object fields (RecordFieldName) - see AvroSchemaBasedParameter/JsonSchemaBasedParameter
   type RecordFieldName = String
   type ParameterName = String
 
-  sealed trait SinkValueParameter {
+  sealed trait SchemaBasedParameter {
```

Comment:
Let's extract this method into a separate file and move it outside of the `sinkvalue` package. Maybe into `parameters` package?

Reply:
Extracted :+1: